recipes/newrelic/infrastructure/super-agent/debian.yml

```diff
@@ -166,14 +176,22 @@ install:
     cleanup:
       cmds:
         - |
-          if [ "{{.NR_CLI_INFRA_AGENT}}" != "false" ] ; then
+          # Log if a previous execution of this recipe was detected, indicating we are skipping this task
+          if [ -f /etc/newrelic-super-agent/.nr-cli ] ; then
+            echo "A previous execution of this recipe was detected. Skipping "cleanup" task."
+          fi
```

Comment:
If we need to skip the whole step, I would use an early return to increase readability
```
if [ -f /etc/newrelic-super-agent/.nr-cli ] ; then
  echo "A previous execution of this recipe was detected. Skipping "config_supervisors" task."
  exit 0
fi
```
I tested it, and it works as I would expect: exits that step and continues in the next one. wdyt?

@pranav-new-relic , do you see any problem with this?

Reply:
👍 makes sense, @rubenruizdegauna. cc @DavSanchez

Reply:
Great suggestion! Implemented in https://github.com/newrelic/open-install-library/pull/1094/commits/0bc100e640a43619d0ce3940344a1e3905708a7c